client/src/components/WorkspaceThumbnailList/index.tsx

```diff
@@ -9,9 +11,14 @@ interface WorkspaceThumbnailListProps {
 }
 
 function WorkspaceThumbnailList({ workspaces }: WorkspaceThumbnailListProps) {
+  const { id: currentId } = useParams();
   const navigate = useNavigate();
+  const [, setWorkspace] = useRecoilState(workspaceState);
```

Comment:
```suggestion
  const setWorkspace = useSetRecoilState(workspaceState);
```
- [useSetRecoilState](https://recoiljs.org/docs/api-reference/core/useSetRecoilState/)

아니면 나중에 workspace 사용할 수 도 있어서 그래요??

Reply:
헉 setState만 가져오는 API가 있는지 몰랐어요. 하지만 #412 에 따라 기각 ...